include/iceflow/contentType.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef CONTENT_TYPE_HPP
+#define CONTENT_TYPE_HPP
+#include <cstdint>
+// ContentType header for defining custom TLV types
+
+namespace ContentType {
+const uint16_t JsonObject = 128; // nlohmann::json object
```

Comment:
Is this actually limited to JSON objects..?

```suggestion
/**
 * JSON
 */
const uint16_t JsonObject = 128; 
```

Reply:
deleting the contentType.hpp. not needed